### ISSUE TITLE
images dataset not included

### ISSUE THREAD

**Issue by shawnFuu** [DekuLiuTesla/CityGaussian#52]
Hi. I really appreciate your work! I try to train the model but I find only colmap dataset included. Could you be so kind to help me find the images dataset or just share it?
Looking forward to your any replies!

**Comment by DekuLiuTesla** [comment]
Hi, @shawnFuu , you can search MegaNeRF and MatrixCity to find the image source.

**Comment by shawnFuu** [comment]
> Hi, @shawnFuu , you can search MegaNeRF and MatrixCity to find the image source.

Thanks a lot for your answer! I find several resources. Which one should I download?
![image](https://github.com/user-attachments/assets/81208cfd-7352-4068-ab52-5bbc80c4576f)

**Comment by Jai132** [comment]
if this is the MatrixCity dataset link then i would suggest you download the small_city/aerial folder rather than the street one

**Comment by shawnFuu** [comment]
> if this is the MatrixCity dataset link then i would suggest you download the small_city/aerial folder rather than the street one

Thanks! Thats helpful.

**Comment by BitCalSaul** [comment]
> if this is the MatrixCity dataset link then i would suggest you download the small_city/aerial folder rather than the street one

I found in this [link](https://github.com/Linketic/CityGaussian/blob/main/doc/data_preparation.md) there should be matrix_city_street in the zip file from google drive. But when I decompress it I found there is no matrix_city_street over there. Could you plz give me some hint? Tks

**Comment by BitCalSaul** [comment]
> if this is the MatrixCity dataset link then i would suggest you download the small_city/aerial folder rather than the street one

May I ask the reason why we download small_city/aerial rather than street?

**Comment by Jai132** [comment]
the link in your comment is for the colmap, its basically the point cloud data for the dataset. the dataset is here at: https://huggingface.co/datasets/BoDai/MatrixCity/tree/main 
and i suggested aerial simply cuz the repo had the config file for aerial and they also had the results already for aerial at: https://huggingface.co/TeslaYang123/CityGaussian/tree/main

**Comment by Jai132** [comment]
if you were referring to the colmap result of street then yes, it wasnt in the zip, i think youll have to run the dataset yourself in colmap software to make it

**Comment by BitCalSaul** [comment]
> the link in your comment is for the colmap, its basically the point cloud data for the dataset. the dataset is here at: https://huggingface.co/datasets/BoDai/MatrixCity/tree/main and i suggested aerial simply cuz the repo had the config file for aerial and they also had the results already for aerial at: https://huggingface.co/TeslaYang123/CityGaussian/tree/main

Thank you very much for your reply. It has been extremely helpful. I am new to large-scale scene training and am looking for a pre-trained model that has been trained on a vast number of scenes (over 10 million or more :) ). So I guess I can find such a model from the second link you provided?

**Comment by Jai132** [comment]
Yes, they did provide large scenes in the link. I think the matrix city aerial and building zip files are the large scale reconstructions with aerial's results being kept separately(lod and normal fine grained) and building's being kept in the same zip
Btw I'm kinda new too, to this stuff😅, got a twitter handle?

**Comment by BitCalSaul** [comment]
> Yes, they did provide large scenes in the link. I think the matrix city aerial and building zip files are the large scale reconstructions with aerial's results being kept separately(lod and normal fine grained) and building's being kept in the same zip Btw I'm kinda new too, to this stuff😅, got a twitter handle?

That's exactly what I was looking for — I've been searching for large-scale scenes with a huge number of Gaussians for quite a while. Really appreciate it, thanks a lot! 🙏 Unfortunately I don't have a Twitter account, but glad to connect here!

**Comment by DYL857857** [comment]
Could you please tell me what the error message is after running the bash script "data_proc_mc.sh"? It seems that there are many missing images in the test set and it doesn't match with the "images.bin" provided by the official version. As a result, the subsequent PSNR test metric is only 14.7. Are there any solutions to this problem?
